vhdbuilder/packer/init-variables.sh

```diff
@@ -213,7 +214,8 @@ cat <<EOF > vhdbuilder/packer/settings.json
   "arm64_sig_resource_group_name": "${ARM64_SIG_RESOURCE_GROUP_NAME}",
   "arm64_sig_gallery_name": "${ARM64_SIG_GALLERY_NAME}",
   "arm64_sig_image_name": "${ARM64_SIG_IMAGE_NAME}",
-  "arm64_sig_image_version": "${ARM64_SIG_IMAGE_VERSION}"
+  "arm64_sig_image_version": "${ARM64_SIG_IMAGE_VERSION}",
+  "allowed_ip": "${PUBLIC_IP}"
```

Comment:
@anujmaheshwari1 allowed IP plumbed here, grabbed from ifconfig.me which is a public service. 

it would be better to know the expected IP in advance, but I think the VM has no IP and this is some Azure NAT'd pub IP so may not be possible. In the future if we adjusted our packer build to run on an agent in one subnet plus build a vm in the same subnet, they could both run without allowed_ip